### PR TITLE
DATAUP-330 test fix

### PR DESCRIPTION
**Pull request by briehl**
This is meant to start fixing front end tests that keep failing on the `truss` branch, due to some rapid dev work.

It's not quite complete, but I think I need help. There's tests that, when the test files are run individually, all pass fine without warnings. But when run in a batch often show console errors. They behave as though they're trying to access mocked fixtures after they've been cleared up, or are continuing to try to execute after the test ends. They might be due to dangling Promises or something happening deep in the stack - specifically in the `fieldTableCellWidget` or the `dynamicDropdownInput`. I can point out the issues in the logs once they run.

Additionally, while trying to check this work in, some form of pre-commit tried to run tests. But they were somehow misconfigured and failed pretty quickly, but prevented me from checking in the changes. I'm not sure how or why that happened, but it was really frustrating, and pre-commit blocking commits with failing tests seems undocumented, and a pretty bad idea.

# Description of PR purpose/changes

* Please include a summary of the change and which issue is fixed. 
* Please also include relevant motivation and context.
* List any dependencies that are required for this change.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
- [ ] Tests pass in travis and locally 
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [ ] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)

# Updating Version and Release Notes (if applicable)

- [ ] [Version has been bumped](https://semver.org/) for each release
- [ ] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

**Comment by briehl** [comment]
Also, related to the test changes, I did some investigation, and the workspace info isn't used or needed anywhere. It's just the workspace id that gets used in various app cell components, so whittled the bulk import cell entrypoint to just use that.

**Comment by ialarmedalien** [comment]
The pre-commit test running was due to an npm module that had snuck back into the `package.json` file -- just make sure that there is no mention of a module called `pre-commit` in package.json or the package lock file; husky now handles all git hooks.

File: test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* global describe, it, expect, spyOn */
+/* global describe it expect spyOn beforeAll afterAll */
```
Comment (ialarmedalien) [inline test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js:1]: Whilst working on eslint/prettier, I found that adding `jasmine` to the env for everything under `tests/` eliminated all these globals.

File: nbextensions/bulkImportCell/main.js [hunk 2]
```diff
@@ -63,27 +62,13 @@ define([
...
-        // TODO where to get config from generally?
-        var workspaceRef = { id: runtime.workspaceId() },
-            workspace = new Workspace(workspaceUrl, {
-                token: runtime.authToken()
-            });
-
-        return workspace.get_workspace_info(workspaceRef);
-
-    }
-
```
Comment (ialarmedalien) [inline nbextensions/bulkImportCell/main.js:77]: 🎉

**Review by eamahanna — APPROVED** [review]
These changes look good to me.

File: test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js (hunk 1 above)
Comment (briehl) [reply test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js:1]: Really? I never got that to work. Might be how I have eslint set up. I'll it.

Comment (ialarmedalien) [reply test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js:1]: Add an .eslintrc.yml file to the test directory, with the content:

```yaml
env:
  amd: true
  es6: true
  jasmine: true
```

and you should be good to go!

Comment (briehl) [reply test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js:1]: Thanks. I added that file.

**Comment by briehl** [comment]
I'll have another addition, but I think this part's ok. Can we merge?

**Pull request merged**